include/TrustWalletCore/TWHRP.h

```diff
@@ -29,6 +30,7 @@ static const char *_Nonnull HRP_BITCOIN = "bc";
 static const char *_Nonnull HRP_BITCOINCASH = "bitcoincash";
 static const char *_Nonnull HRP_LITECOIN = "ltc";
 static const char *_Nonnull HRP_COSMOS = "cosmos";
+static const char *_Nonnull HRP_LUX = "bc";
```

Comment:
it's not here: https://github.com/satoshilabs/slips/blob/master/slip-0173.md

Reply:
We obviously don't want to keep bc as a prefix, so that is/will be subject to change in the future

Reply:
Why do you need to specify then? 

If you specify bech32 - you should use it as the default address for the chain, it's not the case. You might want to register with slip 173 and then make changes?

Reply:
Good point. We can even remove this if necessary, until I don't change the prefix